src/main/java/com/unzer/payment/AbstractPayment.java

```diff
@@ -129,13 +138,15 @@ public void setBasketId(String basketId) {
         this.basketId = basketId;
     }
 
+    @Deprecated
     public Basket getBasket() throws HttpCommunicationException {
         if (basket == null && isNotEmpty(getBasketId())) {
             basket = fetchBasket(getBasketId());
         }
         return basket;
     }
 
+    @Deprecated
```

Comment:
It doesnt look to me thate the properties/methods in this file should be deprecated.
Also it doesnt seem to be relevant for this task.